pandas-stubs/core/indexes/timedeltas.pyi

```diff
@@ -79,12 +79,12 @@ class TimedeltaIndex(DatetimeTimedeltaMixin[Timedelta], TimedeltaIndexProperties
     def shift(self, periods: int = ..., freq=...) -> Self: ...
 
 def timedelta_range(
-    start: TimedeltaConvertibleTypes = ...,
-    end: TimedeltaConvertibleTypes = ...,
-    periods: int | None = ...,
+    start: TimedeltaConvertibleTypes | None = None,
```

Comment:
Docstring says the default is None so I had to change the annotation

https://github.com/pandas-dev/pandas/blob/e4a03b6e47a8ef9cd045902916289cbc976d3d33/pandas/core/indexes/timedeltas.py#L256-L259